### PR TITLE
AT_09.01_001 | My Views>Create new view

### PR DESCRIPTION
**Pull request by DimaCurkanu**
https://trello.com/c/HtUlhnAo/553-at0901001-my-viewscreate-new-view
https://trello.com/c/rtc7RcCb/737-err-homepage-verify-homepage-link-learn-more-about-distributed-builds-is-working
https://trello.com/c/5Gbw4omJ/732-err-homepage-verify-homepage-link-learn-more-about-distributed-builds-is-working

### CODE REVIEW

File: cypress/e2e/homepage.cy.js
Comment (engineeryulia) [inline cypress/e2e/homepage.cy.js:1]: I can see you skip two tests. Just wondering if you file them?

**Review by engineeryulia — APPROVED** [review]
LGTM

File: cypress/e2e/homepage.cy.js
Comment (DimaCurkanu) [reply cypress/e2e/homepage.cy.js:1]: Those two tests messed Maria Moroz to make commit her test during Monday's lesson. And she skips two of four tests. But today two 'it' that дуае in the test failed my CI testing. And I ask advice from Maria and did what she said.

**Review by IrinaVorontsova1 — APPROVED** [review]

File: cypress/e2e/myViewsCreateNewView.cy.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/// <reference types="cypress"/>
+
+describe('My Views>Create new view', () => {
```
[the remark this answers is not in the document]
Comment (IrynaKolh) [reply cypress/e2e/myViewsCreateNewView.cy.js:3]: Появилась идея: возможно вам нужно вынести в beforeEach создание job, а в afrerEach удаление view

[the remark this answers is not in the document]
Comment (AlexSQE) [reply cypress/e2e/myViewsCreateNewView.cy.js:3]: Ирина, про переменные ты наверное имела в виду fixtures.

**Pull request merged**